RocketChat/Rocket.Chat.ReactNative
# [TESTS] Fix some simple text

## Proposed changes
Fix some text. One bash message and a bunch of email addresses that are used in registration.

## Issue(s)	
None.

## How to test or reproduce
- Run `e2e/docker/controlRCDemoEnv.sh startandwait` and see the output
- Open Diego's mailbox and see the amount of spam that the tests are generating :)

## Types of changes


## Checklist
